tests/unit/cloud/__init__.py

```diff
@@ -1,66 +0,0 @@
-"""
-    tests.unit.cloud
-    ~~~~~~~~~~~~~~~~
-"""
-import pytest
-
-import salt.cloud
-from tests.support.unit import TestCase
-
-
-class CloudTest(TestCase):
-    @pytest.mark.slow_test
-    def test_vm_config_merger(self):
```

Comment:
I'm not seeing where this test was ported over to pytest

Reply:
done